packages/amplication-data-service-generator/src/server/create-server.ts

```diff
@@ -65,8 +63,9 @@ export async function createServerModules(
   logger.info("Creating swagger...");
   const swaggerModule = await createSwagger();
 
+  const userEntity = entities.find((entity) => entity.name === "User");
   logger.info("Creating seed script...");
-  const seedModule = await createSeedModule(userEntity);
+  const seedModule = await createSeedModule(userEntity as Entity);
```

Comment:
why do we need this?

Reply:
because the result from userEntity could be undefined

Reply:
![image](https://user-images.githubusercontent.com/39680385/192133304-2977bc9a-110b-4d35-ba33-fa3f84f07878.png)